Source/WebKit/Scripts/webkit/tests/GeneratedSerializers.cpp

```diff
@@ -515,6 +515,44 @@ std::optional<WebCore::FloatBoxExtent> ArgumentCoder<WebCore::FloatBoxExtent>::d
     };
 }
 
+
+void ArgumentCoder<NullableSoftLinkedMember>::encode(Encoder& encoder, const NullableSoftLinkedMember& instance)
+{
+    static_assert(std::is_same_v<std::remove_cvref_t<decltype(instance.firstMember)>, RetainPtr<DDActionContext>>);
+    static_assert(std::is_same_v<std::remove_cvref_t<decltype(instance.secondMember)>, RetainPtr<DDActionContext>>);
+    encoder << !!instance.firstMember;
+    if (!!instance.firstMember)
+        encoder << instance.firstMember;
+    encoder << instance.secondMember;
+}
+
+std::optional<NullableSoftLinkedMember> ArgumentCoder<NullableSoftLinkedMember>::decode(Decoder& decoder)
+{
+    std::optional<RetainPtr<DDActionContext>> firstMember;
+    std::optional<bool> hasfirstMember;
```

Comment:
Could we adjust the code generation to camel-case `hasfirstMember`?

(not at all necessary to land this patch)

Reply:
We could, but you're right that it doesn't matter in this PR.